README.md

```diff
@@ -1,5 +1,345 @@
+website. The following table summarizes the important ways in which those views
+differ from each other.
+
+| Name                       | How to access          | GitHub branch | Vercel env | Storyblok API key | Next.js preview? | Storyblok version param | Display top banner? | Top banner color | Button to enter/exit preview? |
+| -------------------------- | ---------------------- | ------------- | ---------- | ----------------- | ---------------- | ----------------------- | ------------------- | ---------------- | ----------------------------- |
+| Production                 | .com/.org URL          | `main`        | production | public            | off              | `published`             | No                  | n/a              | n/a                           |
+| Storyblok (yellow banner)  | via Storyblok UI       | `develop`     | preview    | preview           | on               | `draft`                 | Yes                 | yellow           | No                            |
+| Vercel URL (green banner)  | via link to Vercel URL | any non-main  | preview    | preview           | off              | `published`             | Yes                 | green            | Yes                           |
+| Vercel URL (yellow banner) | via enter-preview      | any non-main  | preview    | preview           | on               | `draft`                 | Yes                 | yellow           | Yes                           |
+
+Let's take the row labeled "Vercel URL (green banner)." This view is accessed by
+clicking on a Vercel SHA-style URL, which looks like
+https://quansight-labs-SHA-quansight.vercel.app. Typically this URL is found
+either on a GitHub pull request or in Slack. The site served by that URL can be
+built from any branch or commit on GitHub except `main`, which is reserved for
+production. It is built with the "preview" Vercel environment. It does not start
+in Next.js preview mode (though it can switch into it). The preview Vercel
+environment contains the preview Storyblok API key. The site passes
+`version=published` to the Storyblok API. It displays a green banner at the top
+of each page. And the green banner contains a way for the end user to switch in
+and out of the Next.js preview mode.
+
+Note that within Storyblok, there is no button to switch out of preview mode.
+This was done by design because the whole point of the Storyblok UI is to be
+able to preview unpublished content.
+
+## System design decisions
+
+Each website should have only one project in Vercel. Having more than one
+project results in several preview URLs being posted to each pull request on
+GitHub, which makes it hard for reviewers to know which preview URL they should
+review. With only one project per website, you get one preview URL for each
+website posted to the pull request. If it's not clear from the pull request
+title or code which website the PR affects, the author should clarify in the PR
+description.
```

Comment:
We could use the `LLC` and `Labs` Github tags on the PR to indicate the sites that are affected by that PR?